File: iOS/Sample/Sample/ViewController.m
//
//  ViewController.m
//  Sample
//
//  Created by rarnu on 3/12/15.
//  Copyright (c) 2015 YiBan. All rights reserved.
//

#import "ViewController.h"
#import <Utils/Toast.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(IBAction)btnClicked:(id)sender {
    [[[Toast makeText:@"toast"] setGravity:ToastGravityBottom] show];
}

@end

File: Swift/Sample/Sample/ViewController.swift
//
//  ViewController.swift
//  Sample
//
//  Created by rarnu on 1/21/15.
//  Copyright (c) 2015 rarnu. All rights reserved.
//

import UIKit
import Utils

class ViewController: UIViewController {

    var jsonString = "{\"name\":\"aaa\",\"age\":100}";
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        var dc = DemoClass()

        JsonUtils.fillJsonToObject(dc, jsonString: jsonString)
        NSLog("dc: name=\(dc.name!), age=\(dc.age!)")
        NSLog("\(JsonUtils.objectToJsonString(dc))")
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

